### PR TITLE
Create sources, genres,authors and labels schema

### PR DESCRIPTION
**Pull request by worashf**
**I have completed the following tasks. :hand:** 

:heavy_check_mark:  Creates catalog_of_things schema.
:heavy_check_mark:  Created authors, sources, genres and labels tables

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+-- drop database catalo-of-things if it is aready created
+DROP DATABASE IF EXISTS catalog_of_things; 
+ -- create catalog_of_things database
+CREATE DATABASE catalog_of_things;
+
+
+- Drop table genres if exists
```
Comment (Emmykage) [inline db/schema.sql:7]: i think yo forgot to comment this line out

File: db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  -- drop authors table 
+DROP TABLE  IF EXISTS authors;
+
+--create authors table
+
+ CREATE TABLE authors (id  INT GENERATED ALWAYS AS IDENTITY, first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255)  NOT NULL,  CONSTRAINT pk_authors PRIMARY KEY(id)
+ )
+
+ CREATE TABLE music_albums(id INT GENERATED ALWAYS AS IDENTITY, genre_id INT, author_id INT, source_id INT, label_id INT, publish_date DATE, archived BOOLEAN, spotify BOOLEAN)
```
Comment (Emmykage) [inline db/schema.sql]: hey Chuma, it seems like you forgot to reference genre_id, author_id, source_id, label_id as foreign keys

File: db/schema.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+  --drop table labels if exists
+ DROP TABLe IF exists labels
+ -- create table labels 
+   CREATE TABLE labels( id INT GENERATED ALWAYS AS IDENTITY,title VARCHAR(255) NOT NULL,
+    color VARCHAR(100) NOT NULL, CONSTRAINT pk_labels PRIMARY KEY(id)
+  )
+-- Drop table sources if exists
+ DROP TABLe IF exists sources
```
Comment (Emmykage) [inline db/schema.sql:23]: kindly correct typo "TABLe"

**Review by Emmykage — CHANGES_REQUESTED** [review]
nicely done.. everyone

**Comment by worashf** [comment]
Hello I fix the SQL typo error.

**Comment by Emmykage** [comment]
Its ok to merge now.

**Review by Emmykage — APPROVED** [review]
nothing left to say... you can now merge the file

**Pull request merged**